code/modules/antagonists/highlander/highlander.dm

```diff
@@ -7,16 +7,20 @@
 
 /datum/antagonist/highlander/apply_innate_effects(mob/living/mob_override)
 	var/mob/living/L = owner.current || mob_override
-	ADD_TRAIT(L, TRAIT_NOGUNS, "highlander")
-	ADD_TRAIT(L, TRAIT_NODISMEMBER, "highlander")
-	ADD_TRAIT(L, TRAIT_SHOCKIMMUNE, "highlander")
+	ADD_TRAIT(L, TRAIT_NOGUNS, "HIGHLANDER_TRAIT")
+	ADD_TRAIT(L, TRAIT_NODISMEMBER, "HIGHLANDER_TRAIT")
+	ADD_TRAIT(L, TRAIT_SHOCKIMMUNE, "HIGHLANDER_TRAIT")
+	ADD_TRAIT(L, TRAIT_NOFIRE, "HIGHLANDER_TRAIT")
+	ADD_TRAIT(L, TRAIT_NOBREATH, "HIGHLANDER_TRAIT")
 	REMOVE_TRAIT(L, TRAIT_PACIFISM, ROUNDSTART_TRAIT)
 
 /datum/antagonist/highlander/remove_innate_effects(mob/living/mob_override)
 	var/mob/living/L = owner.current || mob_override
-	REMOVE_TRAIT(L, TRAIT_NOGUNS, "highlander")
-	REMOVE_TRAIT(L, TRAIT_NODISMEMBER, "highlander")
-	REMOVE_TRAIT(L, TRAIT_SHOCKIMMUNE, "highlander")
+	REMOVE_TRAIT(L, TRAIT_NOGUNS, "HIGHLANDER_TRAIT")
+	REMOVE_TRAIT(L, TRAIT_NODISMEMBER, "HIGHLANDER_TRAIT")
+	REMOVE_TRAIT(L, TRAIT_SHOCKIMMUNE, "HIGHLANDER_TRAIT")
+	REMOVE_TRAIT(L, TRAIT_NOFIRE, "HIGHLANDER_TRAIT")
+	REMOVE_TRAIT(L, TRAIT_NOBREATH, "HIGHLANDER_TRAIT")
```

Comment:
It's just `HIGHLANDER_TRAIT` not `"HIGHLANDER_TRAIT"`.

You're using the `#define`, rather than a string literal. Means if you misspell the define, the compiler will complain, but if it was a misspelled string literal, it would silently just break.

Reply:
Ye that was just me being dumb... fixed now